<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="8" column="0">
      <widget class="QPushButton" name="button1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="buttonPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="button6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="button7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="buttonDecimal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="buttonSign">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="button8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="button5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="buttonPowerTwo">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>x^2</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="button0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="button2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="buttonClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #F0A30A;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="button4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="buttonParLeft">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="buttonTimes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="2">
      <widget class="QPushButton" name="buttonEquals">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #60A917;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="buttonDiv">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="buttonClearAll">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #E51400;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="buttonMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="button9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLineEdit" name="displayOperations">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: #333;
	border: 1px solid black;
	color: #A4C400;
	padding: 5px;
}</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="buttonParRight">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="buttonPowerAny">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>x^y</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="button3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #647687;
	border-color: 1px solid black;
	padding: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: #aaa;
	border-color: 1px solid black;
	padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="QLineEdit" name="display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto Mono Medium</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: #333;
	border: 1px solid black;
	color: #A4C400;
	padding: 10px;
}</string>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
